scripts/load_script/src/image_load.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import os
+import uuid
+from argparse import Namespace
+from datetime import datetime
+from io import BytesIO
+from typing import List
+
+from minio import Minio
+from requests import get
+
+
+def parse_date(date: str) -> datetime:
+    parsed_date = datetime.strptime(date, "%Y-%m-%d")
+
+    return parsed_date
+
+
+def cli() -> Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('search_request', type=str, help='help')
+    parser.add_argument('bucket', type=str, help='help')
+    parser.add_argument('date', help='help', type=parse_date)
+    list_of_arguments = parser.parse_args()
+
+    return list_of_arguments
+
+
+def get_links(search_request: str, date: datetime) -> List[str]:
+    dog_images_url = {}
+
+    api_url = f'https://imsea.herokuapp.com/api/1?q={search_request} {date}'
+    response = get(url=api_url)
+    url_response = response.json()
+    dog_images_url = url_response['results'][::2]
+
+    return dog_images_url
+
+
+def load_in_bucket(list_of_urls: list, search_request: str, 
+                   bucket_name: str, date: datetime) -> None:
+
+    client = Minio(
+        endpoint='host.docker.internal:9000',
+        secret_key=os.getenv('MINIO_SECRET_KEY'),
+        access_key=os.getenv('MINIO_ACCESS_KEY'),
+        secure=False
+    )
+
+    for link in list_of_urls:
+        response = get(url=link)
+        unique_name = str(uuid.uuid4())
+        img = BytesIO(response.content)
+        size = len(response.content)
+        img.seek(0)
```

Comment:
Here you can proceed without loading the whole image to the memory. I will explain later.